book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals.push(rental)
+    rental.book = self
+  end
+end
```

Comment:
- [ ] good job on setting up the associations 👍  however in order to create the many-to-many (also has-many-through) relationship between `Person` and `Book` using the intermediate class `Rental` the Getter and Setter should be present for `@rentals`
- [ ] also to add a rental in the `add_rental` method the person, book and date should be used as arguments when initiating a new instance of the `Rental` class.

Reply:
The book and date arguments are already taken care of in the initialize method of the rental class. I already set the reader for rentals since you wouldn't want to set a setter for an array but add one rental at a time therefore the add_rental method

Reply:
- [ ] I would also recommend adding Getters and Setters for `@rentals`. Also, in the `add_rental` method you have to create the new rental achieving the `has-many` association. [ Hint: `Rental.new(date, self, person)`] (where book itself should be sent).

For clarification to your above doubt, whatever you have done in `Rental` class is `'belongs-to'` association and in this class you have to set for the `has-many` so you will not repeat anything but it will create a relationship between book and rentals.